musicbot/bot.py

```diff
@@ -3624,6 +3691,78 @@ async def cmd_option(self, player, option, value):
                     )
                 )
 
+    @owner_only
+    async def cmd_cache(self, leftover_args, opt="info"):
+        """
+        Usage:
+            {command_prefix}cache
+
+        Display cache storage info or clear cache files.
+        Valid options are:  info, clear
+        """
+        opt = opt.lower()
+        valid_opt = ["info", "clear"]
+        if opt not in valid_opt:
```

Comment:
From Just Snow: I'd prefer if opt not in ["info, "clear"] then throw an invalid usage error

Reply:
What platform did you develop and test this code on btw?

Reply:
> What platform did you develop and test this code on btw?

This was tested on Ubuntu 22.04 server.  I have not tested on windows and I could not confirm via the python docs and a little search if `getatime()` is a problem for windows or NTFS.  It _should_ work, but I'd have to load a VM to test it.